chipsec/hal/vmm.py

```diff
@@ -163,7 +163,7 @@ def dump_device(self):
         bars = self.cs.pci.get_device_bars(self.bus, self.dev, self.fun)
         for (bar, isMMIO, is64bit, bar_off, bar_reg, size) in bars:
             if isMMIO:
-                chipsec.hal.mmio.dump_MMIO( self.cs, bar, size )
+                chipsec.hal.mmio.MMIO( self.cs ).dump_MMIO( bar, size )
```

Comment:
could the change be self.cs.mmio.dump_MMIO(bar,size) instead?  Would prefer not to create a new object.